### PR TITLE
GPDB DOCS - Fix issues found when validating files in gpdb-webhelp.di…

### PR DESCRIPTION
**Pull request by mkiyama**
…tamap.

Issues are invalid links beween and within dita files.

[ci skip]

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -2642,9 +2632,9 @@
           </thead>
           <tbody>
             <row>
-              <entry colname="col1">integer >= 0</entry>
-              <entry colname="col2">0</entry>
-              <entry colname="col3">master<p>restart</p></entry>
+              <entry colname="col1">integer</entry>
+              <entry colname="col2">64</entry>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: This isn't a validation fix.  Is this change intentional?

File: gpdb-doc/dita/ref_guide/yaml_spec.xml [hunk 2]
```diff
@@ -139,7 +140,7 @@ date: 1865
 <i>       - name</i><i>type</i>
      <xref href="#topic3/OPTIMIZE" format="dita">OPTIMIZE</xref>: STRICT IMMUTABLE
      <xref href="#topic3/MODE" format="dita">MODE</xref>: SINGLE | MULTI</codeblock>
-      <codeblock>  - <xref href="#/topic3/TRANSITION" format="dita">TRANSITION | CONSOLIDATE | FINALIZE</xref>:
+      <codeblock>  - <xref href="#topic3/TCF" format="dita">TRANSITION | CONSOLIDATE | FINALIZE</xref>:
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/yaml_spec.xml]: Same question here - fine if this PR includes some other fixes.  Just want to make sure it's intentional and not reverting some other change.

Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/yaml_spec.xml]: This one is ok. The destination link ID was changed from TRANSITION to TCF

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml (hunk 1 above)
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: No this one is unintended. I'm not sure how but the description of gp_connections_per_thread was reverted to the old version. :-(
